api/test/e2e/sections/sections-crud.spec.ts

```diff
@@ -55,7 +55,7 @@ describe('Page Sections API', () => {
     const res = await testManager
       .request()
       .get(
-        '/sections?filters[0][name]=eu-member-state&filters[0][operator]==&filters[0][values]=Spain&filters[0][values]=Belgium',
+        '/sections?filters[0][name]=eu-member-state&filters[0][operator]==&filters[0][values][0]=Belgium',
```

Comment:
I guess the query() issue you were having is forcing you to use magic strings here.

Please let me know what the issue is, probably we have a fix in place